### PR TITLE
Fix VALIDATE variables name for Dockerfile linters

### LINKED ISSUE #567: VALIDATE_DOCKER env var is not working
**Describe the bug**
Setting the VALIDATE_DOCKER environment variable to true or false does not have the behavior documented in the [README](https://github.com/github/super-linter#environment-variables).
It is similar for VALIDATE_DOCKER_HADOLINT.

**To Reproduce**

1. Run the command
```console
docker run -e RUN_LOCAL=true -v /path-to-folder:/tmp/lint /
-e VALIDATE_DOCKER=true  github/super-linter:latest
```
2. In the logs you see that all languages are still going to be validated:
```console
[INFO  ]   Gathering user validation information...
[DEBUG ]   - Validating [ANSIBLE] files in code base...
[DEBUG ]   - Validating [ARM] files in code base...
[DEBUG ]   - Validating [BASH] files in code base...
[DEBUG ]   - Validating [CLOUDFORMATION] files in code base...
[DEBUG ]   - Validating [CLOJURE] files in code base...
[DEBUG ]   - Validating [COFFEESCRIPT] files in code base...
[DEBUG ]   - Validating [CSS] files in code base...
[DEBUG ]   - Validating [DART] files in code base...
[DEBUG ]   - Validating [DOCKERFILE] files in code base...
...
```

**Expected behavior**
Only DOCKERFILE language should be validated and all others should be excluded.

### PR DESCRIPTION
**Pull request by julie-garrone**
Fixes #567

## Proposed Changes

1. Fix dockerfilelint and Dockerfile Hadolint VALIDATE variables name in code and README.

### Author/Contributor
- [ ] If documentation is needed for this change, has that been included in this pull request

### Reviewing Maintainer 
- [ ] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

File: lib/linter.sh [hunk 1]
```diff
@@ -173,8 +173,8 @@ VALIDATE_CLOJURE="${VALIDATE_CLOJURE}"                               # Boolean t
 VALIDATE_COFFEE="${VALIDATE_COFFEE}"                                 # Boolean to validate language
 VALIDATE_CSS="${VALIDATE_CSS}"                                       # Boolean to validate language
 VALIDATE_DART="${VALIDATE_DART}"                                     # Boolean to validate language
-VALIDATE_DOCKER="${VALIDATE_DOCKER}"                                 # Boolean to validate language
-VALIDATE_DOCKER_HADOLINT="${VALIDATE_DOCKER_HADOLINT}"               # Boolean to validate language
+VALIDATE_DOCKERFILE="${VALIDATE_DOCKERFILE}"                                 # Boolean to validate language
+VALIDATE_DOCKERFILE_HADOLINT="${VALIDATE_DOCKERFILE_HADOLINT}"               # Boolean to validate language
```
Comment (kirintwn) [inline lib/linter.sh:177]: Comment indentation shall be aligned.

File: README.md [hunk 2]
```diff
@@ -200,8 +200,8 @@ But if you wish to select or exclude specific linters, we give you full control
...
-| **VALIDATE_DOCKER**              | `true`                | Flag to enable or disable the linting process of the Docker language.                                                                                                            |
-| **VALIDATE_DOCKER_HADOLINT**     | `true`                | Flag to enable or disable the linting process of the Docker language.                                                                                                            |
+| **VALIDATE_DOCKERFILE**              | `true`                | Flag to enable or disable the linting process of the Docker language.                                                                                                            |
+| **VALIDATE_DOCKERFILE_HADOLINT**     | `true`                | Flag to enable or disable the linting process of the Docker language.                                                                                                            |
```
Comment (kirintwn) [inline README.md:204]: Table indentation shall be aligned.

File: lib/linter.sh (hunk 1 above)
Comment (julie-garrone) [reply lib/linter.sh]: Thanks, it has been fixed.

File: README.md (hunk 2 above)
Comment (julie-garrone) [reply README.md]: Fixed!

**Comment by admiralAwkbar** [comment]
@julie-garrone fixed the test case hopefully...

**Comment by admiralAwkbar** [comment]
@julie-garrone great work and thank you!

**Pull request merged**